webrender/src/renderer.rs

```diff
@@ -4358,8 +4358,14 @@ impl<B: hal::Backend> Renderer<B> {
                 }
             }
         }
-        #[cfg(not(feature = "gl"))]
-        self.device.begin_render_pass(_transit_to_present);
+        #[cfg(not(feature = "gl"))]{
+            if !composite_state.opaque_tiles.is_empty()
```

Comment:
nit: add a helper `is_empty` method to the `composite_state`?

Reply:
Added it.